### PR TITLE
Adding asesurfacefinder

### PR DESCRIPTION
**Pull request by joegilkes**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by joegilkes** [comment]
I confirm that I am willing to be listed as a maintainer.

**Comment by joegilkes** [comment]
@conda-forge/staged-recipes Ready for review!

File: recipes/asesurfacefinder/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{% set name = "asesurfacefinder" %}
```
Comment (ocefpaf) [inline recipes/asesurfacefinder/meta.yaml:1]:
```suggestion
{% set python_min = "3.10" %}
{% set name = "asesurfacefinder" %}
```

File: recipes/asesurfacefinder/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  sha256: 08dd5927a184a361d43a2a52ba2da5f9b0002ba7bea8e096b792d99439f4aa94
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.10
```
Comment (ocefpaf) [inline recipes/asesurfacefinder/meta.yaml:19]:
```suggestion
    - python {{ python_min }}
```

File: recipes/asesurfacefinder/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    - matplotlib-base
+    - sparse >=0.15
+
+test:
+  imports:
+    - asesurfacefinder
+  commands:
+    - pip check
+  requires:
+    - pip
```
Comment (ocefpaf) [inline recipes/asesurfacefinder/meta.yaml:39]:
```suggestion
    - python {{ python_min }}
    - pip
```

File: recipes/asesurfacefinder/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  number: 0
+
+requirements:
+  host:
+    - python >=3.10
+    - setuptools >=75.0.0
+    - wheel
+    - pip
+  run:
+    - python >=3.10
```
Comment (ocefpaf) [inline recipes/asesurfacefinder/meta.yaml:24]:
```suggestion
    - python >={{ python_min }}
```

**Comment by joegilkes** [comment]
Build is failing due to some lazy f-string formatting on my part which is erroring on Python 3.10 (and wasn't previously on Python 3.12). It needs to be fixed but I'll have to push a new version of the pip package and update this PR after. Too late to do that this evening so I'll get it fixed tomorrow.

**Pull request merged**